### DISCUSSION TITLE
Redirecting to Categories

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#35883] [Mobile] [answered]
### Redirecting to categories/repositories is not working well. 

As you see on the below, it can not resolve the repository. 
![IMG_20221012_121331](https://user-images.githubusercontent.com/71941576/195302621-29d90e48-6eef-4439-99ec-b6789f17daf6.jpg)

### How did this happen?

I was reading `README.me` and tapped at hyperlink of categories and this error occurred.

**Comment by iskae** [comment, marked as answer]
Hi @Neodevils 👋 

Oh, that is definitely not the nicest experience 😞 

Would you be able to elaborate a bit further? Can you maybe post the url structure too?

Cheers!

**Reply by Neodevils** [reply]
It's this repository's categories on readme.md file. Where you will be redirected to discussion channels.

**Reply by iskae** [reply]
Oh, thank you so much once again for additional context ❤️ 

This was a legit bug on Android 🙇 I have created an issue and we will be fixing it asap! Please stay tuned for our future releases for the fix 🙇 

Cheers!

**Reply by Neodevils** [reply]
Alright, good luck for the team!
